testsuite/scripts/pom.xml

```diff
@@ -31,6 +31,8 @@
 
     <artifactId>wildfly-testsuite-scripts</artifactId>
 
+    <name>WildFly: Testsuite Scripts</name>
```

Comment:
Please make this "WildFly Test Suite: Scripts"

Reply:
Thanks, @bstansberry I have changed accordingly.